openpathsampling/analysis/tis_analysis.py

```diff
@@ -65,27 +60,22 @@ def all_movers(self):
             all_movers += self.movers[movetype]
         return all_movers
 
-    def _assign_sample(self, sample):
-        if sample.ensemble in self.all_ensembles:
-            try:
-                self._samples_by_ensemble[sample.ensemble].append(sample)
-            except KeyError:
-                self._samples_by_ensemble[sample.ensemble] = [sample]
-            try: 
-                self._samples_by_id[sample.replica].append(sample)
-            except KeyError:
-                self._samples_by_id[sample.replica] = [sample]
+    # TODO: @dwhs can this function be removed?
+    # def _assign_sample(self, sample):
+    #     if sample.ensemble in self.all_ensembles:
+    #         try:
+    #             self._samples_by_ensemble[sample.ensemble].append(sample)
+    #         except KeyError:
+    #             self._samples_by_ensemble[sample.ensemble] = [sample]
+    #         try:
+    #             self._samples_by_id[sample.replica].append(sample)
+    #         except KeyError:
+    #             self._samples_by_id[sample.replica] = [sample]
```

Comment:
It might be in an ipython notebook somewhere. I couldn't find it either -- and I'm not entirely sure what it was supposed to do -- I think it predates your fast-loading of samples, so it's probably not needed. I'd say leave as is: commented out, but not deleted, in case it comes back to bite us somewhere!